.github/workflows/ci-actions-incremental.yml

```diff
@@ -258,7 +258,7 @@ jobs:
             name: "11 Windows",
             java-version: 11,
             maven_args: "-DskipDocs -Dformat.skip",
-            maven_opts: "-Xmx1500m -XX:MaxMetaspaceSize=1g",
+            maven_opts: "-Xmx1500m -XX:MaxMetaspaceSize=1g -Xlog:gc*=debug:stdout",
```

Comment:
Are we sure we don't want to push it to a file and upload it as an artifact? I can do the uploading part if you tweak the syntax to push it to a file :).

Reply:
Done :) I had thought of it, but lazily decided not to use a file since I wasn't sure what directory path to specify for it. In this update, I just specify a file name. I haven't found documentation which states where it will place this file, but I'm guessing it's going to be the current working directory.

Reply:
I pushed an update with `**/windows-java-11.txt` to be on the safe side.

Reply:
The latest run of this PR generated 3 `windows-java-11.txt` files (which is fine since it's one per JVM launch within that run) and all 3 of them have captured the gc logs. It's now a matter of waiting for the failure to occur to get the logs for comparison.